mono/metadata/domain-internals.h

```diff
@@ -608,7 +608,12 @@ MonoJitInfo* mono_jit_info_table_find_internal (MonoDomain *domain, char *addr,
 void mono_enable_debug_domain_unload (gboolean enable);
 
 MonoReflectionAssembly *
-mono_domain_try_type_resolve_checked (MonoDomain *domain, char *name, MonoObject *tb, MonoError *error);
+mono_domain_try_type_resolve_name (MonoDomain *domain, const char *name, MonoError *error);
+
+struct _MonoReflectionTypeBuilder;
+
+MonoReflectionAssembly *
+mono_domain_try_type_resolve_typebuilder (MonoDomain *domain, struct _MonoReflectionTypeBuilder *builder, MonoError *error);
```

Comment:
Use `MonoReflectionTypeBuilder *builder`

Reply:
If it worked I would but I don't think it works.